internal/config/config_test.go

```diff
@@ -202,6 +208,14 @@ func TestIsValid(t *testing.T) {
 			wantErr:    true,
 			wantErrMsg: "unable to parse host mount",
 		},
+		{
+			name: "Invalid config - invalid pull request url",
+			cfg: Config{
+				PullRequest: "123",
```

Comment:
nit: change the string to a url, `https://github.com/owner/repo/issues/123`.

Reply:
done